webclient/projekt/tables.py

```diff
@@ -115,6 +117,15 @@ class Meta:
             "kulturni_pamatka",
         )
 
+    def render_planovane_zahajeni(self, value):
+        if value == "" or value is None:
+            return None
+        if isinstance(value, DateRange):
+            if value.lower and value.upper:
+                format_str = "%Y-%m-%d"
+                return value.lower.strftime(format_str) + " - " + (value.upper - timedelta(days=1)).strftime(format_str)
```

Comment:
```suggestion
                return f"{value.lower.strftime(format_str)} - {(value.upper - timedelta(days=1)).strftime(format_str)}"
```

U tabulek bych preferoval f-stringy kvůli větší efektivitě.

Reply:
Opraveno